packages/dds/tree/src/simple-tree/treeApi.ts

```diff
@@ -82,6 +84,12 @@ export interface TreeNodeApi {
 	 * Returns the {@link TreeStatus} of the given node.
 	 */
 	readonly status: (node: TreeNode) => TreeStatus;
+
+	/**
+	 * Returns the {@link LocalNodeKey} of the given node if the identifier field kind exists.
+	 * Otherwise returns undefined.
+	 */
+	shortID(node: TreeNode): number | undefined;
```

Comment:
There may be some debate about what exactly to name this function in the future, but for now let's at least have it be `shortId` rather than `shortID`. This is consistent without how we case other acronyms - for example, note that the interface that this function belongs to is `TreeNodeApi` rather than `TreeNodeAPI`.

Reply:
Updated with suggestion